vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -48,7 +48,7 @@ type internal FSharpCheckerProvider
     ) =
     let checker = 
         lazy
-            let checker = FSharpChecker.Create()
+            let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = false)
```

Comment:
FWIW I've always been very skeptical about this huge project cache size used by VFPT.  It is much, much, much bigger than that used previously by Visual F# Tools (3 projects).  

I know @vasily-kirichenko thinks differently though - but I just wanted to highlight that this setting doesn't feel instinctively right to me, and my gut intuition is that it will lead to OOM in VS too often.  We need a methodology to test operational memory usage on typical scenarios such as 5-10 very large projects, or hundreds of small projects.

Some information on Roslyn's choices here may also be useful - how much semantic analysis information does Roslyn keep live, and what policies/techniques are used to manage that.